test/rubygems/test_gem_commands_sources_command.rb

```diff
@@ -247,7 +247,7 @@ def test_execute_add_redundant_source_trailing_slash
   end
 
   def test_execute_add_http_rubygems_org
-    http_rubygems_org = 'http://rubygems.org'
+    http_rubygems_org = 'https://rubygems.org/'
```

Comment:
The test is `test_execute_add_http_rubygems_org` please add a new test for `https` - `test_execute_add_https_rubygems_org`